lib/DiscoveryStreamFeed.jsm

```diff
@@ -1547,7 +1555,9 @@ getHardcodedLayout = basic => {
           {
             type: "TopSites",
             header: {
-              title: "Top Sites",
+              title: {
```

Comment:
Title in the layout can be either a localized object or a string. If it's a hard coded string, we must be mindful that it'll show up in de. This is probably fine as hard coded layouts are under our control, and remote layouts are the likely candidates for hard coded strings, which is for testing purposes, and mostly we do test on a per locale basis.